lib/checkPullRequestLabels.js

```diff
@@ -0,0 +1,78 @@
+module.exports.checkProjectLabel = async function(context) {
+  var pullRequest = context.payload.pull_request;
+  var pullRequestNumber = pullRequest.number;
+
+  // eslint-disable-next-line no-console
+  console.log('RUNNING PROJECT LABEL CHECK ON PULL REQUEST ' +
+    pullRequestNumber + ' ...');
+
+  var projectLabel = context.payload.label.name.trim();
+  var hasProjectLabel = projectLabel.toUpperCase().startsWith('PROJECT');
+
+  // If the PR has a project label, assign the project owner
```

Comment:
What if the owner is already assigned?

Also, how do you ensure this comment only comes up once in the PR lifecycle? E.g. the project owner looks at it and approves and then deassigns themselves ... do they get reassigned again?

Reply:
This check will run only when a project label is added to a PR by the author, that is, this is triggered by `pull_request.labelled` hook. So, it will basically run once in the PR lifecycle when the author adds a project label.

Reply:
OK.

What happens if there are two project labels added to a PR?

Reply:
I tested this just now -- the bot will assign all of the project owners according to the label. Each label triggers a separate event on which the bot acts.